users/engler/fairseq/training.py

```diff
@@ -232,6 +236,82 @@ def run(self):
             my_env["PYTHONPATH"] = self.fairseq_root
         sp.check_call(self._get_run_cmd(), env=my_env)
 
+    def plot(self):
+        directory = "./outputs"
+        train_score, train_error = {}, {}
+        valid_score, valid_error = {}, {}
+        learning_rates = {}
+
+        for cur in os.walk(directory):
+            dir_path = cur[0]
+            files = cur[2]
+            if "hydra_train.log" in files:
+                with open(f"{dir_path}/hydra_train.log", "r") as f:
+                    lines = f.readlines()
+                    for i in range(len(lines)-1):
+                        line = lines[i]
+                        if 'begin validation on "valid" subset' in line:
+                            split = re.sub('[{,":]', "", lines[i+1]).split(" ")
```

Comment:
```suggestion
                            split = re.sub('[{,":]', "", lines[i + 1]).split(" ")
```
I really wonder why this is not flagged by black...